### PR TITLE
migration component role

### PR DESCRIPTION
**Pull request by plepaisant**
### Description

The compoent has been migrated in components

Test cases have been created

### Example 

```jsx
// Sample code

//check one role
 <Role  hasOne={['DEFAULT_ROLE']}} >
{children}
</Role>

//check many roles all are needed
 <Role  hasAll={['DEFAULT_ROLE', 'ADMIN_ROLE', 'OTHER_ROLE']} >
{children}
</Role>

//optional child element when roles are not sufficient
 <Role  hasOne={['DEFAULT_ROLE']}}  emptyBlock={<div>Insufficient rights</div>}>
{children}
</Role>

```

### CODE REVIEW

File: src/components/role/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const React = require('react/addons');
```
Comment (pierr) [inline src/components/role/__tests__/index.js]: You don't need to require this.

File: src/components/role/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    <TestWrapper mockedValue={childBlock}>
+                        <div>Test</div>
+                    </TestWrapper>
+                </Role>
+            </TestWrapper>
+        );
+    }
+}
+
+describe.only('The role', () => {
```
Comment (pierr) [inline src/components/role/__tests__/index.js]: remove only

File: src/components/role/__tests__/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    describe('when user has not the role(s)', () => {
+
+        before(
+            () => {
+                user.setRoles(['DEFAULT_ROLE', 'ADMIN_ROLE', 'OTHER_ROLE']);
+            }
+        );
+        it('user has not the role ', () => {
+            const renderedTest = TestUtils.renderIntoDocument(<TestComponent rolesForHasOne={['USER_ROLE']} />);
+            expect(() => TestUtils.findRenderedDOMComponentWithClass(renderedTest, childBlock)).to.throw(CHILD_NOT_FOUND_MSG);
```
Comment (pierr) [inline src/components/role/__tests__/index.js]: `const  {renderIntoDocument,findRenderedDOMComponentWithClass } = TestUtils`

**Comment by plepaisant** [comment]
Done

**Pull request merged**